agent.go

```diff
@@ -39,9 +40,10 @@ var processed_ids []string
 var responses = "RESPONSE_CHANNEL"
 var registration = "REGISTRATION_CHANNEL"
 var commands = "COMMANDS_CHANNEL"
-var bearer = "Bearer " + "BEARERTOKEN"
+var bearer = "BEARERTOKEN"
 var token = "TOKENTOKEN"
-var key = []byte("AESKEY")
+var key = "AESKEY"
```

Comment:
When injecting at compile-time, the variable has to be a `string`, while the AES code needs `[]byte`, so I created the `keyBytes` variable immediately after it.